test/unit/acknowledgement/process-invalid.test.js

```diff
@@ -0,0 +1,242 @@
+const mockCommit = jest.fn()
+const mockRollback = jest.fn()
+const mockTransaction = {
+  commit: mockCommit,
+  rollback: mockRollback
+}
+
+jest.mock('../../../app/data', () => {
+  return {
+    sequelize:
+       {
+         transaction: jest.fn().mockImplementation(() => {
+           return { ...mockTransaction }
+         })
+       }
+  }
+})
+
+jest.mock('../../../app/config')
+const mockConfig = require('../../../app/config')
+
+jest.mock('../../../app/event')
+const mockEvent = require('../../../app/event')
+
+jest.mock('../../../app/acknowledgement/get-hold-category-name')
+const getHoldCategoryName = require('../../../app/acknowledgement/get-hold-category-name')
+
+jest.mock('../../../app/holds/get-hold-category-id')
+const getHoldCategoryId = require('../../../app/holds/get-hold-category-id')
+
+jest.mock('../../../app/reschedule/index')
+const holdAndReschedule = require('../../../app/reschedule/index')
+
+jest.mock('../../../app/reset/index')
+const { resetPaymentRequestById } = require('../../../app/reset/index')
+
+const mockFRN = require('../../mock-frn')
+const processInvalid = require('../../../app/acknowledgement/process-invalid')
+const { SFI } = require('../../../app/schemes')
+
+let mockAcknowledgement
+let mockAcknowledgementError
+let mockHoldCategoryName
+let mockHoldCategoryId
```

Comment:
We never assign this let variable which should map onto the `holdCategoryId` of `'DAX rejection'` for the `SFI` scheme (search the `changelog` for `DAX rejection`)

Reply:
Do you mean let mockHoldCategoryId never gets assigned?

Reply:
Yeah `let mockHoldCategoryId`

Inside `beforeEach` we should assign it a value